Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js

```diff
@@ -86,6 +86,10 @@ WI.ConsoleSnippetTreeElement = class ConsoleSnippetTreeElement extends WI.Script
 
         contextMenu.appendSeparator();
 
+        contextMenu.appendItem(WI.UIString("Delete"), () => {
```

Comment:
this should probably be named "Delete Console Snippet" to match the "Run Console Snippet"

Reply:
@dcrousso Makes sense. This patch landed before I could address this. May I have a R+ on this follow-up?
https://github.com/rcaliman-apple/WebKit/pull/new/eng/Web-Inspector-Add-context-menu-item-to-delete-a-saved-Console-Snippet-follow-up